### ISSUE TITLE
Add a streamsx.plumbing repository

### ISSUE THREAD

**Issue by scotts** [IBMStreams/administration#52]
The streamsx.plumbing repository will contain operators that change the flow of tuples in an application, but are not part of the logic of the application. I would like to add ElasticLoadBalance to this repository initially, which is a drop-in replacement for ThreadedSplit, but with elasticity.

However, I see other operators falling under this category. For example, I have encountered several implementations of merges, to be used in conjunction with UDP, for preserving tuple ordering in parallel regions. We also have several examples from our own Standard Toolkit which fall under this category (such as Gate, Switch and Throttle), and may be more appropriate here than in the generic "utility" category.

**Comment by petenicholls** [comment]
Creating streamsx.plumbing.

**Comment by petenicholls** [comment]
done.
